pkg/apis/camel/v1/camelcatalog_types.go

```diff
@@ -75,6 +75,7 @@ type CamelCatalogStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Runtime Version",type=string,JSONPath=`.spec.runtime.version`,description="The Camel K Runtime version"
 // +kubebuilder:printcolumn:name="Runtime Provider",type=string,JSONPath=`.spec.runtime.provider`,description="The Camel K Runtime provider"
+// +kubebuilder:resource:path=camelcatalogs,scope=Namespaced,shortName=cc,categories=kamel;camel
```

Comment:
Sorry I may not have be enough specific. The `kubebuilder:resource` marker already exists few lines above (74). Only the `categories` should be appended to the existing marker, instead of duplicating it.

Reply:
,categories=kamel;camel

> LGTM

you are more than welcome and thank you for the review